docs/nightly/en/reference/sql/data-types.md

```diff
@@ -64,6 +64,126 @@ Output:
 | `TimestampMillisecond` | 64-bit timestamp values with milliseconds precision | 8 Bytes |
 | `TimestampMicroSecond` | 64-bit timestamp values with microseconds precision | 8 Bytes |
 | `TimestampNanosecond` | 64-bit timestamp values with nanoseconds precision | 8 Bytes |
+| `Interval`|  Time interval | 4 Bytes for `YearMonth`, 8 Bytes for `DayTime` and 16 Bytes for `MonthDayNano`|
+
+### Interval Type
+
+`Interval` type is used in scenarios where you need to track and manipulate time durations.It can be written using the following verbose syntax:
+
+```
+ quantity unit [quantity unit...]
+```
+
+* `quantity`:  is a number (possibly signed),
+* `unit`:   is `microsecond`, `millisecond`, `second`, `minute`, `hour`, `day`, `week`, `month`, `year`, `decade`, `century`, or abbreviations or plurals of these units; 
```

Comment:
```suggestion
QUANTITY UNIT [QUANTITY UNIT...]
```

* `QUANTITY`:  is a number (possibly signed),
* `UNIT`:   is `microsecond`, `millisecond`, `second`, `minute`, `hour`, `day`, `week`, `month`, `year`, `decade`, `century`, or abbreviations or plurals of these units; 
```